src/LazyLoad.jsx

```diff
@@ -60,10 +64,11 @@ LazyLoad.propTypes = {
   children: PropTypes.node.isRequired,
   height: PropTypes.oneOfType([
     PropTypes.string,
-    PropTypes.number,
```

Comment:
Ok, no problem, I can remove it. Other than that, this look ok to merge after I put the commas back?